### PR TITLE
Update to 0.2.0

### PR DESCRIPTION
**Pull request by leouieda**
The bot didn't seem to pick this up so I'm doing it manually. Had to tweak the package URL since we had some trouble with the PyPI release and were forced to add a `-1` to it. 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 552fb15217de85736dc7fea045feccc4c5a4bec3d6b25e617124f6fd633c288f
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-1.tar.gz
+  sha256:
```
Comment (ocefpaf) [inline recipe/meta.yaml:10]: We need a checksum here. Also I'm not sure that ver-1 is a pep440 valid version. Maybe we should raise that upstream.

**Comment by leouieda** [comment]
Ah right, I forgot to paste the hash...

@santisoler maybe we should just release 0.2.1 just to fix these things.

**Comment by santisoler** [comment]
>  Also I'm not sure that ver-1 is a pep440 valid version. Maybe we should raise that upstream.

The `-1` is not part of the version number, but the number of the build uploaded to PyPI. I had to re upload the build because of a minor mistake, so I followed https://github.com/pypa/packaging-problems/issues/74#issuecomment-693987158 . But if this is problematic, we can release `v0.2.1` and publish that one.

File: recipe/meta.yaml (hunk 1 above)
Comment (santisoler) [inline recipe/meta.yaml:10]:
```suggestion
  sha256: 01a84f5249644dd373d2e9bc5b3fd91819bfbe77ec275d4f829d70a508252f44
```

**Comment by leouieda** [comment]
Let's merge this and see what happens. If there is a problem we can make a new release.

**Pull request merged**